dev/run-tests-jenkins.py

```diff
@@ -165,12 +165,6 @@ def main():
     if "test-maven" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_TOOL"] = "maven"
     # Switch the Hadoop profile based on the PR title:
```

Comment:
BTW while you're at it, I spotted one more thing that we could remove although it probably won't matter at all. Look for `expect_equal(basenameSansExtFromUrl(y), "spark-2.1.0-bin-hadoop2.4-without-hive")` in `test_utils.R`